app/node/nodeHelpers.ts

```diff
@@ -15,46 +15,26 @@ import {
 import { PeerElement } from '../proto/concordium_p2p_rpc_pb';
 import {
     AccountInfo,
-    Account,
     Global,
     Fraction,
     TransactionStatus,
 } from '../utils/types';
 
-export interface AccountInfoPair {
-    account: Account;
-    accountInfo: AccountInfo;
-}
-
 export async function getlastFinalizedBlockHash(): Promise<string> {
     const consensusStatus = await getConsensusStatus();
     return consensusStatus.lastFinalizedBlock;
 }
-/** Gets the accountInfos for each given accounts. returns a list of objects
- *   each containing an account and its accountInfo.
- */
-export async function getAccountInfos(
-    accounts: Account[]
-): Promise<AccountInfoPair[]> {
-    const blockHash = await getlastFinalizedBlockHash();
-    const accountInfos: AccountInfoPair[] = await Promise.all(
-        accounts.map(async (account) => {
-            const accountInfo = await getAccountInfo(
-                account.address,
-                blockHash
-            );
-            return { account, accountInfo };
-        })
-    );
-    return accountInfos;
-}
 
 /** Gets the accountInfo for the given address. */
 export async function getAccountInfoOfAddress(
     address: string
 ): Promise<AccountInfo> {
     const blockHash = await getlastFinalizedBlockHash();
-    return getAccountInfo(address, blockHash);
+    const accountInfo = await getAccountInfo(address, blockHash);
+    if (!accountInfo) {
+        throw new Error(`Address:${address} does not represent an account.`);
```

Comment:
No `accountInfo` can also be because the node is not caught up, so I don't think the error message is correct.

Reply:
I have changed the error message.